pimd/pim_cmd.c

```diff
@@ -3415,15 +3415,25 @@ static void igmp_show_groups(struct pim_instance *pim, struct vty *vty, bool uj)
 	time_t now;
 	json_object *json = NULL;
 	json_object *json_iface = NULL;
-	json_object *json_row = NULL;
+	json_object *json_group = NULL;
+	json_object *json_groups = NULL;
 
 	now = pim_time_monotonic_sec();
 
-	if (uj)
+	if (uj) {
 		json = json_object_new_object();
-	else
+		json_object_int_add(json, "Total Groups",
+				    pim->igmp_group_count);
+		json_object_int_add(json, "Watermark limit",
+				    pim->igmp_watermark_limit);
+	} else {
+		vty_out(vty, "Total IGMP groups: %u\n", pim->igmp_group_count);
+		vty_out(vty, "Watermark warn limit(%s) : %u\n",
```

Comment:
space after `limit`

Reply:
Confused a bit. 
Correlated with prev line and understood as removing space after ')' and removed it.
Did you mean to give space after 'limit'  before '(' ? Then will change it accordingly.